ocaml/xapi/monitor_types.ml

```diff
@@ -37,7 +37,6 @@ let vif_device_of_string x =
 let find_rrd_files prefix =
```

Comment:
Should this now be `let find_rrd_files ()`?

Reply:
indeed!